src/application.ts

```diff
@@ -79,6 +80,7 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
     const defaultLogLimitBlocks: number = parseInt(DEFAULT_LOG_LIMIT_BLOCKS) || 10000
     const aatPlan = AAT_PLAN || AatPlans.PREMIUM
     const redirects: string | object[] = REDIRECTS || ''
+    const commitHash: string | string = COMMIT_HASH || ''
```

Comment:
If for some reason the commit hash fails, what about just replacing it with a quick random string instead of blank. Since this is only set when the application boots, it would have the same basic effect.

Reply:
Hm true.